src/components/ColorPicker/ColorPicker.visual.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { VisualTestContainer } from '../../../test/visual/VisualTestContainer';
+import { ColorPicker } from './';
+
+class ColorPickerVisual extends React.Component<any> {
+  render() {
+    return (
+      <VisualTestContainer>
+        <ColorPicker {...this.props} onChange={e => {}}>
+          <ColorPicker.Item aria-label={'red color'} value={'red'} />
```

Comment:
Instead of having only one color item, lets add this to the props below 👇 .
We can then have tests for several colors, and a selected color as well.

Reply:
done